File: linear_search/linear_search.js
function linearSearch(array, number) {
    return array.indexOf(number) !== -1 ? true : false;
}

function manualLinearSearch(array, number) {
    let numberFound = false;
    for(let i = 0; i < array.length; i++) {
        if(array[i] === number) {
            numberFound = true
        }
    }

    return numberFound;
}

let arr = [1,2,3,4,5];

console.log(linearSearch(arr, 4));
console.log(linearSearch(arr, 6));

File: linear_search/linear_search.py
from typing import List
def linear_search(arr: List[int], num: int):
    found_number = False
    try:
        arr.index(num)
        found_number = True
        return found_number
    except:
        return found_number

def manual_linear_search(arr: List[int], num: int):
    found_number = False
    for number in arr:
        if number == num:
            found_number = True
    
    return found_number

arr: List[int] = [1,2,3,4,5]
print(linear_search(arr, 4))
print(linear_search(arr, 6))
print(manual_linear_search(arr, 4))
print(manual_linear_search(arr, 6))